packages/components/src/components/tab-group/tab-group.ts

```diff
@@ -303,6 +306,30 @@ export default class SdTabGroup extends SolidElement {
     this.hasScrollControls = this.nav.scrollWidth > this.nav.clientWidth;
   }
 
+  private async repositionIndicator() {
+    await this.updateComplete;
+
+    const currentTab = this.activeTab;
+    const indicator = this.renderRoot.querySelector<HTMLElement>('#indicator');
+    const tabsContainer = this.renderRoot.querySelector<HTMLElement>('[part="tabs"]');
```

Comment:
I think it's best to move this query to `firstUpdated` or instead use a getter function.

[https://lit.dev/docs/components/shadow-dom/#accessing-nodes-in-the-shadow-dom](https://lit.dev/docs/components/shadow-dom/#accessing-nodes-in-the-shadow-dom)